src/identity.js

```diff
@@ -628,27 +629,29 @@ export class Identity extends EventEmitter {
 
     /**
      * Generates the link to the new login page that'll be used in the popup or redirect flow
-     * @param {string} state - An opaque value used by the client to maintain state between the
+     * @param {object} options
+     * @param {string} [options.state] - An opaque value used by the client to maintain state between the
```

Comment:
This should be `options.state` without the square brackets `[]` (they tell jsdoc that an argument is Optional, and state is Required..)

Reply:
Uhm.. no.. but it's "optional with no default value", so:
```javascript
/**
@param {string} someRequiredArg
@param {string} [optionalArg]
@param {string} [optionalArgWithDefault='someDefault']
@param {string} requiredWithDefault=123
*/
```